### DISCUSSION TITLE
Blackhole / Reaper / Skype

### DISCUSSION THREAD

**Discussion by mooch99** [ExistentialAudio/BlackHole#132] [General]
Trying to route audio from Reaper to Skype and while Reaper is getting a signal, Skype will not see it.  Pretty sure I have my routing set up correctly.  Instead of using the built-in input and output I am using my Scarlett 2i2 as my soundcard since that is what my mic and guitar are plugged into.  I have attached screenshots of my configuration.  What am I doing wrong?
![Screen Shot 2020-05-05 at 3 23 11 PM](https://user-images.githubusercontent.com/64843229/81107200-c9599380-8ee4-11ea-84e3-c984a48f3678.png)
![Screen Shot 2020-05-05 at 3 23 26 PM](https://user-images.githubusercontent.com/64843229/81107203-cb235700-8ee4-11ea-822f-e9a15c9dfeec.png)
![Screen Shot 2020-05-05 at 3 23 45 PM](https://user-images.githubusercontent.com/64843229/81107206-cc548400-8ee4-11ea-91c3-31bdfbc8b217.png)
![Screen Shot 2020-05-05 at 3 22 43 PM](https://user-images.githubusercontent.com/64843229/81107212-ce1e4780-8ee4-11ea-80f2-9d63b380bf33.png)
![Screen Shot 2020-05-05 at 3 25 18 PM](https://user-images.githubusercontent.com/64843229/81107216-cf4f7480-8ee4-11ea-85f2-f14d636c52b0.png)

**Comment by devinroth** [comment]
You need to setup Reaper to send to the aggregate channels 3-4 (for the scarlett) and 5-6 (for blackhole). It looks like you just don't have the routing in Reaper setup properly.
